pykube/objects.py

```diff
@@ -12,6 +12,9 @@
 
 
 class ObjectManager:
+    #function all
```

Comment:
these comments don't make sense, Python has docstrings for this

Reply:
indeed, I forgot to clean them up. Sorry about that.